apps/omg_api/lib/application.ex

```diff
@@ -23,6 +23,7 @@ defmodule OMG.API.Application do
 
   def start(_type, _args) do
     deposit_finality_margin = Application.fetch_env!(:omg_api, :deposit_finality_margin)
+    exiters_finality_margin = Application.fetch_env!(:omg_api, :exiters_finality_margin)
```

Comment:
I see your point, but this is dangerous. Note that you've missed one spot where you should add the new `exiters_finality_margin` config variable (in the root `config/config.exs`, where you'll find `deposit_finality_margin`. 

After you add that, I think the best way to make it safe is to make a runtime check **here** which raises an error if `deposit_finality` <= `exiters_finality`